src/main/java/hello/controller/CustomerController.java

```diff
@@ -20,12 +21,26 @@
     @Autowired
     private CustomerRepository respository;
 
+    static Logger log = Logger.getLogger(CustomerController.class);
+
 
     @PostMapping
-    public ResponseEntity addCustomer(@RequestBody Customer customer) { //to turn json object in java customer
+    public ResponseEntity addCustomer(@RequestBody Customer customer){ //to turn json object in java customer
+        System.out.println(customer);
+        if ((customer.getName().isEmpty()) || (customer.getName() == null)) {
+            log.debug("Hello World");
+
+            throw new BadRequestException("Customer name needs to be entered");
+
+    }if (customer.isBookingConfirmed() == null) {
+            throw new BadRequestException("booking request cannot be empty");
```

Comment:
If name is null or empty and isBookingConfirmed is empty only one error will be return to the client. Is there a way around that?